tests/python/test_numpy.py

```diff
@@ -215,11 +215,26 @@ def func2(a: ti.types.ndarray()):
         assert n[i, j, k] == i + j + k
 
 
-@test_utils.test(require=ti.extension.data64)
-def test_numpy_i64_index():
-    @ti.kernel
-    def foo(a: ti.types.ndarray(), i: ti.i64) -> ti.i64:
-        return a[i]
+@test_utils.test()
+def test_numpy_op_with_matrix():
+    scalar = np.cos(0)
+    arr = np.array([0, 1])
+    vec = ti.Vector([1, 2])
+    arr2d = np.array([[0, 1], [2, 3]])
+    mat = ti.Matrix([[0, 1], [2, 3]])
 
-    x = np.array([1, 2, 3])
-    assert foo(x, 1) == 2
+    assert isinstance(scalar + vec, ti.Matrix) and isinstance(
+        mat + arr2d, ti.Matrix)
+
+    @ti.kernel
+    def test():
```

Comment:
https://docs.taichi-lang.org/docs/reference#the-assert-statement 
IIRC assert only works when debug=True?

Reply:
Thanks for this detailed review! I have added `debug=True` in the test.